gm4_metallurgy/data/gm4_metallurgy/functions/repair_skulls.mcfunction

```diff
@@ -1,32 +1,29 @@
-
-execute if entity @s[type=item,nbt={Item:{tag:{SkullOwner:{Properties:{textures:[{Value:"eyJ0aW1lc3RhbXAiOjE0ODE0MTExMzQwMDQsInByb2ZpbGVJZCI6Ijk4NWIyNzVlYmI1YTQzNDRiNDM3Njg5NTI4NjNhNjNmIiwicHJvZmlsZU5hbWUiOiJTcGFya3MiLCJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvMWRkZjMzZDViYmMxZmJjYTA0MDIzZjNkNWJjNjhkMWFkODRjN2RjYmUxZTJkYTk3YzIxOGVjOWNiZDQ0YTUifSwiQ0FQRSI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzVjM2NhN2VlMmE0OThmMWI1ZDI1OGQ1ZmE5MjdlNjNlNDMzMTQzYWRkNTUzOGNmNjNiNmE5Yjc4YWU3MzUifX19"}]}}}}}] run data merge entity @s {Item:{tag:{gm4_metallurgy:{item:"ore",ore_type:"thorium"},CustomModelData:20,display:{Name:'{"translate":"%1$s%3427655$s","with":["Thorianite Lump",{"translate":"item.gm4.metallurgy.thorianite_lump"}],"italic":false,"color":"white"}',Lore:['[{"translate":"%1$s%3427655$s","with":["Contains ",{"translate":"text.gm4.metallurgy.contains"}],"italic":false,"color":"gray"},{"translate":"%1$s%3427655$s","with":["Thorium",{"translate":"text.gm4.metallurgy.thorium"}],"italic":false,"color":"dark_gray"}]']}}}}
-
+# run from item
+
+# repair ores
+# copper ore
+execute if entity @s[type=item,nbt={Item:{tag:{SkullOwner:{Properties:{textures:[{Value:"eyJ0aW1lc3RhbXAiOjE0ODE0MTA4Mjk2OTAsInByb2ZpbGVJZCI6Ijk4NWIyNzVlYmI1YTQzNDRiNDM3Njg5NTI4NjNhNjNmIiwicHJvZmlsZU5hbWUiOiJTcGFya3MiLCJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvNDNjMWE3ZGEyZWJkZGI0YTk3YzI3NDA0N2RlNzFkN2RjZWJjMTc2YTBlNjE3MzBmZmQzMzIzODUzNTNlIn0sIkNBUEUiOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS81YzNjYTdlZTJhNDk4ZjFiNWQyNThkNWZhOTI3ZTYzZTQzMzE0M2FkZDU1MzhjZjYzYjZhOWI3OGFlNzM1In19fQ=="}]}}}}}] run data merge entity @s {Item:{tag:{gm4_metallurgy:{item:"ore",metal:{type:"copper",castable:1b}},CustomModelData:18,display:{Name:'{"translate":"%1$s%3427655$s","with":["Malachite Lump",{"translate":"item.gm4.metallurgy.malachite_lump"}],"italic":false,"color":"white"}',Lore:['[{"translate":"%1$s%3427655$s","with":["Contains ",{"translate":"text.gm4.metallurgy.contains"}],"italic":false,"color":"gray"},{"translate":"%1$s%3427655$s","with":["Copper",{"translate":"text.gm4.metallurgy.copper"}],"italic":false,"color":"gold"}]']}}}}
```

Comment:
are these fix functions supposed to have the amount in them? Right now all I'm seeing is `metal:{type:"copper",castable:1b}`

Reply:
Yup they should! Good spot, thanks!